src/components/v5/common/ActionSidebar/partials/ActionSidebarLoadingSkeleton/ActionSidebarLoadingSkeleton.tsx

```diff
@@ -0,0 +1,231 @@
+              <LoadingSkeleton
+                isLoading
+                className={clsx('h-5 w-full rounded', {
+                  'max-w-[6.25rem]': index !== 1 && index !== 3,
+                  'md:max-w-[5.625rem]': index === 1,
+                  'md:max-w-[5rem]': index === 3,
+                })}
+              />
+            </div>
+            <div className="mt-2 flex w-1/2 items-center gap-2">
+              {index !== 2 && index !== 3 && (
+                <LoadingSkeleton
+                  isLoading
+                  className={clsx('h-5 w-full max-w-[18.75rem] rounded', {
+                    'rounded-3xl': index === 1 || index === 5,
+                    'max-w-[8.125rem]': index === 0,
+                    'max-w-[18.75rem]': index === 6,
+                    'max-w-[5.6875rem]': index === 1 || index === 5,
+                    'max-w-[6.875rem]': index === 4,
+                    'max-w-[4.5625rem]': index === 2,
+                  })}
+                />
+              )}
+              {index === 2 && (
+                <div className="flex items-center gap-2">
+                  <LoadingSkeleton isLoading className="h-5 w-5 rounded-3xl" />
+                  <LoadingSkeleton
+                    isLoading
+                    className="h-5 w-[2.8125rem] rounded"
+                  />
+                </div>
+              )}
+              {index === 3 && (
+                <div className="flex items-center gap-2">
+                  <LoadingSkeleton
+                    isLoading
+                    className="h-5 w-[2.8125rem] rounded"
+                  />
+                  <LoadingSkeleton
+                    isLoading
+                    className="ml-1 h-5 w-5 rounded-3xl"
+                  />
+                  <LoadingSkeleton
+                    isLoading
+                    className="h-5 w-[2.8125rem] rounded"
+                  />
+                </div>
+              )}
+            </div>
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+
+  if (isCreateActionSkeleton) {
+    return <div>{renderSkeleton()}</div>;
+  }
+
+  return (
+    <div className="flex flex-grow flex-col-reverse justify-end overflow-auto sm:flex-row sm:justify-start">
+      {renderSkeleton()}
+
+      <div
+        className={`
+              w-full
+              border-b
+              border-b-gray-200
+              bg-gray-25
+              px-6
+              py-8
+              sm:h-full
+              sm:w-[23.75rem]
+              sm:flex-shrink-0
+              sm:overflow-y-auto
+              sm:border-b-0
+              sm:border-l
+              sm:border-l-gray-200
+            `}
+      >
+        <div className="flex">
+          <div className="mr-[-.3450rem] mt-2">
+            {new Array(72).fill(0).map(() => (
+              <LoadingSkeleton
+                isLoading
+                className="mb-[.0625rem] h-[.1875rem] w-[.0625rem] rounded"
+              />
+            ))}
```

Comment:
Nice job figuring this out. It feels like there should be a better way of doing this, but I can't think of one, so nice solution. 👍